frontend/www/js/omegaup/components/user/Dependents.vue

```diff
@@ -25,11 +40,43 @@
 <script lang="ts">
 import { Vue, Component, Prop } from 'vue-property-decorator';
 import T from '../../lang';
+import * as ui from '../../ui';
 import { types } from '../../api_types';
 
 @Component
 export default class UserDependents extends Vue {
   @Prop() dependents!: types.UserDependentsPayload[];
+  @Prop() userVerificationDeadline!: Date | null;
+
   T = T;
+  ui = ui;
+
+  get bannerColor() {
+    const today = new Date();
+    const deadline = new Date(this.userVerificationDeadline as Date);
+    if (deadline.toDateString() === today.toDateString()) {
```

Comment:
Si estas dos variables sólo las utilizas para determinar el color que se van a remarcar, ¿Puedes usar la propiedad computada `daysUntilVerificationDeadline`?

Se me ocurre algo así:
```suggestion
    if (this.daysUntilVerificationDeadline <= 1) {
```
De esa forma ya no utilizarías las constantes `deadline` y `today` en esta función

Reply:
Si es verdad, gracias